services/apps/alcs/src/alcs/home/home.controller.ts

```diff
@@ -91,8 +91,8 @@ export class HomeController {
       const reconsiderations =
         await this.reconsiderationService.getBy(assignedFindOptions);
 
-      const planningReviews =
-        await this.planningReviewService.getBy(assignedFindOptions);
+      // const planningReviews =
```

Comment:
I assume this is one of those comments that will addressed once feature is ready

Reply:
Yes, there is a separate ticket for home page requirements that this will be addressed in.